src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,54 @@
 'use strict';
 
+const DEFAULT_HEALTH = 100;
+const BITE = 50;
+
 class Animal {
-  // write your code here
+  constructor(name, health = DEFAULT_HEALTH) {
+    this.name = name;
+    this.health = health;
+
+    if (!Animal.alive) {
+      Animal.alive = [];
+    }
```

Comment:
You don't need to do it every time when an instance of an animal is created, only once. So you can move `Animal.alive = []` outside of class